corehq/apps/reports/views.py

```diff
@@ -1104,6 +1120,22 @@ def _render_report_configs(request, configs, domain, owner_id, couch_user, email
     return response.content.decode("utf-8"), excel_attachments
 
 
+def _is_location_restricted(domain, config, couch_user):
+    '''
+    Checks if request is location restricted for the asked report.
+    Returns True if request is restricted for the asked user and the report. Returns False otherwise.
+    '''
+    if not toggles.LOCATION_RESTRICTED_SCHEDULED_REPORTS.enabled(domain):
```

Comment:
Looks like [this](https://github.com/dimagi/commcare-hq/pull/32964#discussion_r1314512777) was missed in this version

Reply:
Moved the condition.
https://github.com/dimagi/commcare-hq/compare/981a3ec8f8270adb2e8a33fcfa0f2a61b9242143..4aa8f3e56bf1b2a71519ce86107f8f83de12182f